doc/errorcodes.rst

```diff
@@ -0,0 +1,80 @@
+.. _errorcodes:
+
+Error codes
+=================
+
+New error codes are frequently added, as the tool is still a work in progress.
+Error codes are documented in the code as well as they are added.
+
+Errors codes can have ``_w`` attached, which denotes the annotation should only be reported as a warning.
+
+For categories retrieved from Ritmyndir, see `here <https://bin.arnastofnun.is/gogn/storasnid/ritmyndir/>`__.
+Other categories currently covered are:
+
+
++------------+----------------------------------------------------------------------------------------------+
+| | Error code                          | Description                                                       |
++------------+----------------------------------------------------------------------------------------------+
+|| ``A001``  |  Abbreviation corrected.                                                                     |
++------------+----------------------------------------------------------------------------------------------+
+|| ``A002``  |  Token corrected as an acronym.                                                              |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C001``  |  Duplicated word removed. Should be corrected.                                               |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C002``  |  Wrongly compounded words split up. Should be corrected.                                     |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C003``  |  Wrongly split compounds united. Should be corrected.                                        |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C004``  |  Duplicated word marked as a possible error. Should be pointed out but not deleted.          |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C005``  |  Possible split compound, depends on meaning/PoS chosen by parser.                           |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C006``  |  A part of a compound word is wrong.                                                         |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C007``  |  A multiword compound such as "skóla-og frístundasvið" correctly split up                    |
++------------+----------------------------------------------------------------------------------------------+
+|| ``E001``  |  Unable to parse sentence, possibly something wrong with it.                                 |
```

Comment:
"Unable to parse sentence. This means that the sentence does not conform to GreynirCorrect's built-in grammar rules. Either the sentence is grammatically deficient, or it is correct but the grammar doesn't cover its structure."

Reply:
Much better description, updated.